docs/synconomicon/src/ffi.md

```diff
@@ -0,0 +1,3 @@
+# FFI
+
+All libraries expose a C-compatible [foreign function interface](https://docs.rs/ffi-support/0.1.3/ffi_support/) for platform-specific bindings.
```

Comment:
This seems out of place with the rest of the synconomicon content, and may be better suited to more general-purpose documentation for the rust-components work as a whole.